pulsar_metrics/metrics/drift.py

```diff
@@ -116,16 +111,17 @@ def evaluate(
         """
 
         try:
+            if self._feature_name is not None:
+                ref_column = reference[self._feature_name]
+                self._column = current[self._feature_name]
+            else:
+                ref_column = reference
+                self._column = current
 
             if self._name != "CvM":
```

Comment:
My comment 
    Define NAME_TYPE = "CvM "
    Variable =  DriftTestMetricsFuncs[self._name].value(self._column, ref_column, **kwargs)
    if self._name != NAME_TYPE :
                _, pvalue = Variable 
            else:
                res = Variable

Reply:
Corrected. The if statement was no longer needed.